modules/cnf-verifying-performance-setting.adoc

```diff
@@ -0,0 +1,279 @@
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true
+  workloadHints:
+    highPowerConsumption: true
+    perPodPowerManagement: false
+    realTime: true
+----
++
+The kernel parameters optimized for absolute minimal latency and maximum determinism at the cost of increased power consumption are as follows: 
++
+[source,terminal]
+----
+BOOT_IMAGE=(hd0,gpt3)/ostree/rhcos-47993a4e9cbc331c977d2e7e68eb1ac348d54079ea10a2b53f27826502b17fd6/vmlinuz-5.14.0-284.54.1.rt14.339.el9_2.x86_64 ignition.platform.id=metal ostree=/ostree/boot.0/rhcos/47993a4e9cbc331c977d2e7e68eb1ac348d54079ea10a2b53f27826502b17fd6/0 root=UUID=958b268c-dd33-4b23-901d-93f034f7df14 rw rootflags=prjquota boot=UUID=4e1006c0-5e24-4b84-841d-e1923c2968a2 skew_tick=1 tsc=reliable rcupdate.rcu_normal_after_boot=1 nohz=on rcu_nocbs=3-15 tuned.non_isolcpus=00000007 systemd.cpu_affinity=0,1,2 intel_iommu=on iommu=pt isolcpus=managed_irq,3-15 nohz_full=3-15 tsc=reliable nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11 processor.max_cstate=1 intel_idle.max_cstate=0 idle=poll intel_pstate=disable systemd.unified_cgroup_hierarchy=0 systemd.legacy_systemd_cgroup_controller=1
+----
++
+.Example 3 
++
+With the following per-pod power management performance profile applied to a single-node OpenShift cluster.
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  cpu:
+    isolated: 3-15
+    reserved: 0-2
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/master: ""
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  numa:
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true
+  workloadHints:
+    highPowerConsumption: false
+    perPodPowerManagement: true
+    realTime: true
+----
++
+The kernel parameters that allow for power management per pod are as follows: 
++
+[NOTE]
+====
+For more information about enabling power savings for a node, see the "Optional: Power saving configurations" section. 
+====
++
+[source,terminal]
+----
+BOOT_IMAGE=(hd0,gpt3)/ostree/rhcos-47993a4e9cbc331c977d2e7e68eb1ac348d54079ea10a2b53f27826502b17fd6/vmlinuz-5.14.0-284.54.1.rt14.339.el9_2.x86_64 ignition.platform.id=metal ostree=/ostree/boot.0/rhcos/47993a4e9cbc331c977d2e7e68eb1ac348d54079ea10a2b53f27826502b17fd6/0 root=UUID=958b268c-dd33-4b23-901d-93f034f7df14 rw rootflags=prjquota boot=UUID=4e1006c0-5e24-4b84-841d-e1923c2968a2 skew_tick=1 tsc=reliable rcupdate.rcu_normal_after_boot=1 nohz=on rcu_nocbs=3-15 tuned.non_isolcpus=00000007 systemd.cpu_affinity=0,1,2 intel_iommu=on iommu=pt isolcpus=managed_irq,3-15 nohz_full=3-15 tsc=reliable nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11 intel_pstate=passive systemd.unified_cgroup_hierarchy=0 systemd.legacy_systemd_cgroup_controller=1
+----
++
+This table describes the list of kernel arguments with a description and guidance on which tuned profile applies those settings:
```

Comment:
I like this table, but it's in the wrong place
Suggest it should be moved up to the section `Understanding workload hints` where it could complement (or replace) the current table